rr_mapper/config/rr_mapper.yaml

```diff
@@ -0,0 +1,29 @@
+# @file rr_mapper.yaml
+# @author Sirui Song
+# @author Jungwook Lee
+# @author Raymond Kuo
+# @author Toni Ogunmade(oluwatoni)
+# @competition IARRC 2018
+
+#Obstacle#
+LaserMapper/INFLATE_OBS:                    8
+LaserMapper/LASER_ORIENTATION:              -1
+
+#Map Parameters#
+#map_res in meter
+LaserMapper/map_res:                        0.01
```

Comment:
Is this resolution the same as the one we were calculating yesterday for lane detection? If so it is gonna be different between all the races and seems like we gotta move this into a higher level params file or just in a upper launch file where both mapper and lane detection can get it from